content/en/docs/troubleshooting/ClusterResourcePlacementEviction.md

```diff
@@ -0,0 +1,260 @@
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionNotExecuted
+    status: "False"
+    type: Executed
+```
+
+In this cae the Eviction object reached a terminal state, it's status has `Executed` condition set to `False`, because
+the `ClusterResourcePlacementDisruptionBudget` object is invalid. For `ClusterResourcePlacement` objects of type 
+`PickAll`, when specifying a `ClusterResourcePlacementDisruptionBudget` the `minAvailable` field should be set to an 
+absolute number and not a percentage and the `maxUnavailable` field should not be set since the total number of 
+placements is non-deterministic.
+
+### Eviction blocked by specified CRPDB
+
+Example status for `ClusterResourcePlacementEviction` object blocked by a `ClusterResourcePlacementDisruptionBudget` 
+object,
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-21T22:01:57Z"
+    message: Eviction is valid
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionValid
+    status: "True"
+    type: Valid
+  - lastTransitionTime: "2025-04-21T22:01:57Z"
+    message: 'Eviction is blocked by specified ClusterResourcePlacementDisruptionBudget,
+      availablePlacements: 2, totalPlacements: 2'
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionNotExecuted
+    status: "False"
+    type: Executed
+```
+
+In this cae the Eviction object reached a terminal state, it's status has `Executed` condition set to `False`, because 
+the `ClusterResourcePlacementDisruptionBudget` object is blocking the eviction.
+
+Taking a look at the `ClusterResourcePlacementDisruptionBudget` object,
+
+```
+arvindthirumurugan@Arvinds-MacBook-Pro kubefleet % kubectl get crpdb pick-all-crp -o YAML
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacementDisruptionBudget
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"placement.kubernetes-fleet.io/v1beta1","kind":"ClusterResourcePlacementDisruptionBudget","metadata":{"annotations":{},"name":"pick-all-crp"},"spec":{"minAvailable":2}}
+  creationTimestamp: "2025-04-21T21:55:44Z"
+  generation: 1
+  name: pick-all-crp
+  resourceVersion: "2748"
+  uid: efc73250-b6dd-4e2d-8387-e7d0e9b5328d
+spec:
+  minAvailable: 2
+```
+
+We can see that the `minAvailable` is set to `2`, which means that at least 2 placements should be available for the 
+`ClusterResourcePlacement` object.
+
+Taking a look at the clusters connected to the Fleet,
+
+```
+kubectl get cluster -A
+NAME             JOINED   AGE   MEMBER-AGENT-LAST-SEEN   NODE-COUNT   AVAILABLE-CPU   AVAILABLE-MEMORY
+kind-cluster-1   True     32m   43s                      2            20750m          15440136Ki
+kind-cluster-2   True     32m   59s                      2            20750m          15440136Ki
+```
+
+We see that there are 2 clusters connected to the Fleet.
+
+Let's take a look at the `ClusterResourceBinding` objects for the `ClusterResourcePlacement` object,
```

Comment:
Hi Arvind! For this part it might be better to include at least a reference telling people to check for availability and act accordingly.